src/languages/es.js

```diff
@@ -33,7 +33,7 @@ export default {
         settings: 'Configuración',
         termsOfService: 'Términos de servicio',
         dob: 'Fecha de Nacimiento',
-        ssn: 'Número de Seguridad Social',
+        ssnLast4: 'Últimos 4 dígitos de su SSN',
```

Comment:
NAB: It was called `Número de Seguridad Social` and we changed it to `SSN`, is that cool ?